maintainers/maintainer-list.nix

```diff
@@ -2057,6 +2057,11 @@
     github = "jlesquembre";
     name = "José Luis Lafuente";
   };
+  jlogemann = {
```

Comment:
I would recommend using your github name here.

Reply:
sure, thank you!